examples/England-Organization-Organisation-Example.xml

```diff
@@ -0,0 +1,67 @@
+<Organization xmlns="http://hl7.org/fhir">
+    <id value="England-Organization-Organisation-Example" />
```

Comment:
Adding a prefix (England-Organization-NHSOrganisation-Example") before the second organization may be required or outrightly use the example being described. In this case GPPractice. (England-Organization-GPPractice-Example")

Reply:
Pushed change